specs/regional_communities.md

```diff
@@ -1,61 +1,333 @@
+- Users can request core memberships
+- Admins can approve core memberships
+- Admins can warn, silence and ban users
+
+### Phase 3: Enable voting
+- Members can request a vote on splitting the Community
+- Members can launch simple EasyPolls
+- Members can vote in a new core user
+- Members can vote to warn / silence / ban an offending user
+
+
 ## Use Cases
 
-### Actors
+## Actors
+<!-- List all collections that will use the widget. -->
+- Members
+   - Admin members
+   - Core members
+   - Home members
+   - Hangaround members
+- Non-members
+   - Non-member humans
+   - Non-member users
+
+## Actions
+<!-- List the actions each collective will take individually. -->
+
+### Find and Join a Community
+Finding a community aligned with my interests should be very easy. It's a core concept of this widget and serves to get people into open and collaborative communities rather than creating competing factions.
+
+Searching for communities should be possible through a few different ways
+- Search by map. Zoom into a map and update the list of available communities accordingly
+- Seach-as-you-type.
+   - Search by fuzzy search of geographies. Start typing in a city, country or region and all available options are shown. 
+   - Search by fuzzy search of constellations. Start typing in a constellation and all available options are shown.
+- Select you language, which updates the available communities. Default language is English.
+- Any combination of the above will further narrow the search
+
+
+### Approve a new Community member
+Approvind a new community member could be as simple as a "follow back" by the owner of the community or someone who has been given the keys to control the community account. 
+
+It is recommended that only human-verified accounts be allowed to become members, as a minimum requirement.
+
+A drawback of this approach is that there is no vote. Any single admin can follow or unfollow, which may be too arbitrary for some communities. Another drawback is that the "follow" feature would no longer be available to use for its originally intended purpose. A more structured approach to membership may have to be contemplated. Optins include:
+- user follows the Community and automatically becomes a member as long as some criteria are filled (e.g. is human verified and accepts the code of conduct) => Preferred
+- user applies for membership, followed by a majority vote to approve the application
+- membership is open to anyone without approval
+- membership is open to anyone, with the restriction that they can be a full member of one and only one community
+
+This author recommends a combination, whereby there are four types of membership:
+1. Home Membership - each user can define their one and only "home" membership = the community that is most closely aligned with them. This can be done without approval from the community, but each user can only select exactly one home community (or not make the selection).
```

Comment:
ditto, home can have many dimensions.

Reply:
Sure. But if home was exclusive, then you could get extra benefits if you made this community your home. Maybe you assign three: one geography home, one language, and one tag that's your fav tag. Something that shows your dedication to a cause. "You can't be an admin in our community unless it aligns with at least two of your three home selections"

Anyways. It was a thought. Maybe a bad idea.